.github/workflows/ci-dgraph-tests.yml

```diff
@@ -60,9 +60,15 @@ jobs:
           # build the test binary
           cd t; go build .
           # run the tests
-          ./t --coverage=true
+          ./t --coverage=true --skip=restore,multi-tenancy,minio
           # clean up docker containers after test execution
           ./t -r
+          ./t --pkg=restore
```

Comment:
change to FQP (fully qualified path)

Reply:
fixed in https://github.com/dgraph-io/dgraph/pull/8351/commits/3d9f7ff0ab02b4e057598a3d7139b1a4f356650f